go/src/koding/klient/machine/mount/index/index.go

```diff
@@ -12,6 +12,9 @@ import (
 	"github.com/djherbis/times"
 )
 
+// Version stores current version of index.
+const Version = 1
```

Comment:
I'm not going to set this on build time nor should we - this won't be changed along with kd version. I added this field only to provide a way to support incompatible changes in the future(git's index is versioned in the same way).